pkgs/development/tools/pipenv/default.nix

```diff
@@ -39,7 +39,11 @@ in buildPythonApplication rec {
 
   propagatedBuildInputs = runtimeDeps;
 
-  doCheck = false;
+  doCheck = true;
+  checkPhase = ''
+    export HOME=$PWD
+    $out/bin/pipenv install ${fetchPypi {pname="pyjokes"; version="0.6.0"; sha256="08860eedb78cbfa4618243c8db088f21c39823ece1fdaf0133e52d9c56e981a5";} }
```

Comment:
Could you replace the whole `${fetchPypi ...}` part with `${wheel.src}`. For that one we already have all the meta set up.

Reply:
With 

```diff
-    $out/bin/pipenv install ${fetchPypi {pname="pyjokes"; version="0.6.0"; sha256="08860eedb78cbfa4618243c8db088f21c39823ece1fdaf0133e52d9c56e981a5";} }
+    $out/bin/pipenv install ${wheel.src}
```

I get the following error:

```bash
running install tests
Creating a virtualenv for this project…
Pipfile: /private/var/folders/ck/dlscb24n4lzdlzpt3w0pdcgh0000gn/T/nix-build-pipenv-2018.11.26.drv-0/pipenv-2018.11.26/Pipfile
Using /nix/store/9a5bl4gvzdm69fvhwbwln5ld6vn55iyx-python3-3.7.5/bin/python3.7 (3.7.5) to create virtualenv…
⠇ Creating virtual environment...Already using interpreter /nix/store/9a5bl4gvzdm69fvhwbwln5ld6vn55iyx-python3-3.7.5/bin/python3.7
Using base prefix '/nix/store/9a5bl4gvzdm69fvhwbwln5ld6vn55iyx-python3-3.7.5'
New python executable in /private/var/folders/ck/dlscb24n4lzdlzpt3w0pdcgh0000gn/T/nix-build-pipenv-2018.11.26.drv-0/pipenv-2018.11.26/.local/share/virtualenvs/pipenv-2018.11.26-kbPUpQ_H/bin/python3.7
Also creating executable in /private/var/folders/ck/dlscb24n4lzdlzpt3w0pdcgh0000gn/T/nix-build-pipenv-2018.11.26.drv-0/pipenv-2018.11.26/.local/share/virtualenvs/pipenv-2018.11.26-kbPUpQ_H/bin/python
Installing setuptools, pip, wheel...
done.
Running virtualenv with interpreter /nix/store/9a5bl4gvzdm69fvhwbwln5ld6vn55iyx-python3-3.7.5/bin/python3.7

✔ Successfully created virtual environment! 
Virtualenv location: /private/var/folders/ck/dlscb24n4lzdlzpt3w0pdcgh0000gn/T/nix-build-pipenv-2018.11.26.drv-0/pipenv-2018.11.26/.local/share/virtualenvs/pipenv-2018.11.26-kbPUpQ_H
Creating a Pipfile for this project…
Installing /nix/store/snai9k7dzscb4qpn7fbizspx7zbdfirv-wheel-0.33.6-source…
error: could not create 'wheel.egg-info': Permission denied
```